common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -64,6 +64,10 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
     }
   }
 
+  public static void warmUpJVM() {
```

Comment:
Please javadoc what this function intends to do, and mark that there might be no guarantee.